src/modules/companies/lib/invoice-accounts.js

```diff
@@ -0,0 +1,146 @@
+const waterServiceRepos = require('../../../lib/connectors/repos');
```

Comment:
I think the code in here should probably be merged with `src/modules/billing/services/invoice-account.js` and placed in the global location:

`src/lib/services`

Although if creating other entities (e.g. company, address) these should belong in the services specific to those entities.